engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -141,26 +137,26 @@ public List<User> getUsers(String accountId, String orgId, boolean adminsOnly) {
         users = transformToUser(usersTotal);
 
         // Micrometer doesn't like when tags are null and throws a NPE.
-        String accountIdTag = accountId == null ? "" : accountId;
-        getUsersTotalTimer.stop(meterRegistry.timer("rbac.get-users.total", "accountId", accountIdTag));
-        getRbacUsersGauge(accountIdTag).set(users.size());
+        String orgIdTag = orgId == null ? "" : orgId;
```

Comment:
Do we have cases where the `orgId` is actually `null` ?

Reply:
It's not supposed to happen now but it still can because the `org_id` field isn't required yet in the notification schema.